yb-voyager/cmd/importSchema.go

```diff
@@ -110,9 +111,39 @@ func importSchema() {
 	}
 
 	log.Info("Schema import is complete.")
+	if isMViewsPresent {
+		utils.PrintAndLog("Materialised Views are present in the imported schema, use --refresh-mviews flag with --post-import-data to refresh mviews after importing data.\n\n")
+	}
+	if flagPostImportData && flagRefreshMViews  {
+		refreshMViewsPostImportData(conn)
+	}
 	callhome.PackAndSendPayload(exportDir)
 }
 
+func refreshMViewsPostImportData(conn *pgx.Conn) {
+	utils.PrintAndLog("\nRefreshing Materialised Views..\n\n")
+	var listOfMViewsImported []string
+	schemaDir := filepath.Join(exportDir, "schema")
+	importMViewFilePath := utils.GetObjectFilePath(schemaDir, "MVIEW")
+	if !utils.FileOrFolderExists(importMViewFilePath) {
+		return
+	}
+	mviewsSqlInfoArr := createSqlStrInfoArray(importMViewFilePath, "MVIEW")
+	for _, eachMviewSql := range mviewsSqlInfoArr {
+		if strings.Contains(eachMviewSql.stmt, "CREATE MATERIALIZED VIEW") {
+			listOfMViewsImported = append(listOfMViewsImported, eachMviewSql.objName)
+		}
+	}
+	log.Infof("List of Mviews Imported to refresh - %v", listOfMViewsImported)
+	for _, eachMView := range listOfMViewsImported{
+		queryRefreshMViews := fmt.Sprintf("REFRESH MATERIALIZED VIEW %s", eachMView)
+		_, err := conn.Exec(context.Background(), queryRefreshMViews)
+		if err != nil {
+			utils.ErrExit("error in refreshing the materialised view %s: %v\n",eachMView, err)
```

Comment:
I don't think you need to add newline character with `ErrExit`. Confirm it, i think its already taken care by internal function calls.